ethereum_bridge/src/parameters.rs

```diff
@@ -218,6 +218,15 @@ impl EthereumBridgeConfig {
             token_cap: DenominatedAmount { amount: cap, denom },
         } in erc20_whitelist
         {
+            if addr == native_erc20
+                && denom != &NATIVE_MAX_DECIMAL_PLACES.into()
+            {
+                panic!(
+                    "Error writing Ethereum bridge config: The native token \
+                     should have {NATIVE_MAX_DECIMAL_PLACES} decimal places"
+                );
+            }
```

Comment:
It would be good to have some validation for this in genesis setup to catch this before a chain is finalized. However, the genesis is currently being heavily reworked in #685 so it might be better to add after

Reply:
https://github.com/anoma/namada/issues/1787